Tests/CacheAdvanceTests/TestableMessage.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Created by Dan Federman on 12/20/19.
+//  Copyright © 2019 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+struct TestableMessage: Codable, CustomStringConvertible, Equatable {
```

Comment:
I think you want [ExpressibleByStringLiteral](https://developer.apple.com/documentation/swift/expressiblebystringliteral) here instead of `CustomStringConvertible`?

That way, you don't need to change these https://github.com/dfed/CacheAdvance/pull/19/commits/8697adf1142c8c0cab58002828478518dbdaea33#diff-57e5815eda868a2058fc13cd1e7dbb46R220